cli/src/commands/git/push.rs

```diff
@@ -966,25 +971,29 @@ fn find_bookmarks_targeted_by_revisions<'a>(
 ) -> Result<Vec<(&'a RefName, LocalAndRemoteRef<'a>)>, CommandError> {
     let mut revision_commit_ids = HashSet::new();
     if use_default_revset {
-        // remote_bookmarks(remote=<remote>)..@
-        let workspace_name = workspace_command.workspace_name();
-        let expression = RevsetExpression::remote_bookmarks(
-            StringPattern::everything(),
-            StringPattern::exact(remote),
-            None,
-        )
-        .range(&RevsetExpression::working_copy(workspace_name.to_owned()))
-        .intersection(&RevsetExpression::bookmarks(StringPattern::everything()));
+        // Get the default revset specified by the user.
+        let push_revset = workspace_command
+            .settings()
+            .get_string("revsets.'git-push(remote)'")?;
```

Comment:
Shouldn't this be `revsets.git-push`? The parameter name can't be changed by user, and `git-push()` (with dashes) isn't a valid function name.

Reply:
The implementation should be the same. I just mean it seems confusing to use fake function-like config key because the key isn't parsed as a function.